src/views/ContributionAssistant.vue

```diff
@@ -197,7 +210,7 @@ export default {
     reloadPage(){
       window.location.reload()
     },
-    setProof(event) {
+    async setProof(event) {
```

Comment:
we don't have much/any async/await in the code elsewhere.
just saying it'll be slightly harder to maintain if we start having both in the code.
can wait a further refactor